index.html

```diff
@@ -272,5 +272,23 @@ <h3 class="sub_title">
         </ul>
       </section>
     </main>
+    <!-- contact form section  -->
+
+    <section class="contact-form">
+     <div class="container">
+      <div class="contact-text">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, 
+          a feature that you need built or a project that needs coding.
+           I’d love to help with it </p>
```

Comment:
I suggest you fix the styling of your form to match the figma design. As part of  this milestones requirement, you are supposed to : "You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)."
This is the figma design:
<img width="376" alt="Screenshot 2022-11-01 at 15 09 18" src="https://user-images.githubusercontent.com/48546992/199230064-4e83d874-bb88-4480-b739-505707d4669e.png">

Kidly fix the following in your design:
<img width="442" alt="Screenshot 2022-11-01 at 15 09 46" src="https://user-images.githubusercontent.com/48546992/199231715-fc568d0c-d09e-428b-a918-81a57250ef3a.png">

- [ ] Currently, your form is rectangular while figma design is shaped differently
- [ ] The figma design has a bottom `<border> or <hr> or <img>` you can use any tag that urs is missing
- [ ] Also your form inputs have a red border while figma are just white
- [ ] Your content `the p and h2` should have a bit of `padding or width`
- [ ] Your button is too far from the bottom of the page.